roles/splunk/tasks/configure_os.yml

```diff
@@ -20,3 +20,9 @@
 - name: Enable read for dmesg
   include_tasks: configure_dmesg.yml
   when: configure_dmesg
+
+- name: Configure firewall service
+  include_tasks: "configure_firewall.yml"
+  when:
+    - firewall_service != 'undefined'
```

Comment:
Why do we need this if we are already checking the `configure_firewall` var?

Reply:
If someone unsets the firewall_service variable manually it should fail before getting to the playbook.